src/user-guide/installing-workflows.rst

```diff
@@ -0,0 +1,358 @@
+
+Excluding Items From Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, cylc install will exclude `.git`, `.svn` directories.
+To configure excluded files and directories from the file installation,
+create a `.cylcignore` file in your source directory, this supports
+pattern matching.
+
+For example:
+We will look at running the cylc install command inside the directory
+`~/cylc-sources/test-flow` with the following directory structure:
+
+.. code-block:: bash
+         
+   $ pwd
+   /home/cylc-sources/test-flow
+
+.. code-block:: bash
+
+   $ tree -all
+   ├── .cylcignore
+   ├── dir1
+   │   ├── another-file
+   │   └── file
+   ├── dir2
+   │   ├── another-file
+   │   └── file
+   ├── file1
+   ├── file2
+   ├── file3
+   ├── flow.cylc
+   ├── textfile1.txt
+   └── textfile2.txt
+
+We wish to omit any files matching the pattern ``*.txt``,  the file 
+``file1``, the contents of ``dir1`` and the contents of ``dir2`` including the
+directory itself.
+
+.. code-block:: bash
+
+   $ cat .cylcignore
+   *.txt
+   file1
+   dir1/*
+   dir2
+
+
+Now we are ready to install our workflow.
+      
+.. code-block:: bash
+
+   $ cylc install
+   INSTALLED test-flow from home/cylc-sources/test-flow -> home/cylc-run/test-flow/run1
+
+Looking at the directory structure that has been created
+
+.. code-block:: bash
+
+   $ tree -all home/cylc-run/test-flow/run1
+   ├── dir1
+   ├── file2
+   ├── file3
+   ├── flow.cylc
+   ├── log
+   │   └── install
+   │       └── <time-stamp>-install.log
+   └── .service
+
+
+.. _Symlinking of Directories:
+
+2. Symlinking of Directories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Upon running ``cylc install``, symlinks for the directories ``run``, ``log``,
+``share``, ``share/cycle`` and ``work`` will be created in accordance with
+the symlink rules for ``localhost`` as defined in
+:cylc:conf:`global.cylc[symlink dirs]`. 
+
+This is overridable via the command line option ``--no-symlinks``, where the 
+directories will not be symlinked.
+
+
+Automatically Generated Directories and Files
+---------------------------------------------
+
+Running ``cylc install`` will generate some extra files in your workflow run
+directory. 
+
+- The :term:`service directory` will be created in preparation for running the 
+  workflow. This is needed to store essential files used by Cylc. 
+
+- A `_cylc-install` directory containing a `source` symlink to the
+  :term:`source directory`.
+  This is needed to enable Cylc to determine the original workflow source
+  for ``cylc reinstall``.
+
+- A new `install` directory in the workflow's log directory, with a
+  time-stamped install log file containing information about the installation.
+
+If a compatible version of Cylc-Rose is installed, ``cylc install`` will
+generate additional files/directories. 
+
+- A directory ``opt`` containing a file ``rose-suite-cylc-install.conf`` 
+
+- A ``rose-suite.conf`` file will also be created in the workflow run
+  directory.
```

Comment:
I think we can omit the Rose information here if we just link through to the `cylc-rose` page:

```suggestion
Cylc plugins e.g. :ref:`cylc-rose` may generate additional files.
```

Will need to add a `cylc-rose` label at the top of [that page](https://github.com/cylc/cylc-doc/blob/master/src/plugins/cylc-rose.rst).